data-structures.md

```diff
@@ -81,6 +81,38 @@ type BlockHeader struct {
 } representation tuple
 ```
 
+## Ticket
+
+A ticket is a shared random value stapled to a particular block in the chain. Every miner must produce a new ticket each time they run a leader election attempt. In that sense, every new block produced will have one or more associated tickets (in the case the block took multiple leader election attempts to produce).
+
+```go
+type Ticket struct {
```

Comment:
please update this to the new IPLD schema syntax

Reply:
Holding off on specifying methods and the like for now, if that is what is meant...